### ISSUE TITLE
swap solid and default header layout roles

### ISSUE THREAD

**Issue by shoulders** [joomdev/jd_seattle#18]
when i first used JD seattle (from scratch) I found it hard to workout why the content was always pushed up. It was becasue the default layout has the header with the absolute posistioning and the whole layout relies on there being content published to the slider or banner area.

When end uses start to use components which are not specified via menu areas or have content in the areas specified they will find their content shoved up underneath the menu and might not figure out why.

My solution is easy.

1) set the solid header as the default template/layout. The website will behave as normal and as people expect.
2) set the current menu items to specify the use of the 'modified header' layout. This also show the user/developer that some non-standard styling has been done and where to look.

**Comment by Nishtha003** [comment]
You are right! We also thought of doing this the right way, therefore we have already implemented your first solution when you try to install the **Template Only** file there we have the solid header as the default template & if anybody wants to make the header transparent they can do that from the **Settings** file.

And as far as the second solution is concerned, we can make that as one of the menu items & keep solid as the default. Will work on this in the coming update hopefully!
